R/interp.R

```diff
@@ -79,25 +123,22 @@ interp <- function(f, data = NULL) {
 
 #' @export
 #' @rdname interp
-UQ <- function(x, data = NULL) {
+UQ <- function(x) {
+  x
+}
+#' @export
+#' @rdname interp
+UQE <- function(x) {
   if (is_formula(x)) {
-    if (is_null(data)) {
-      f_rhs(interp(x))
-    } else {
-      f_eval(x, data = data)
-    }
+    f_rhs(x)
   } else {
     x
   }
 }
-
 #' @export
 #' @rdname interp
 UQF <- function(x) {
-  if (!is_formula(x)) {
-    abort("`x` must be a formula")
-  }
-  x
+  bquote(`_F`(.(x)))
```

Comment:
Seems a little weird to be using `bquote()` here - maybe just construct the call directly?